databricks_cli/runs/cli.py

```diff
@@ -106,12 +112,32 @@ def list_cli(api_client, job_id, active_only, completed_only, offset, limit, out
       - Result state (can be n/a)
     """
     check_version(api_client, version)
-    runs_json = RunsApi(api_client).list_runs(
-        job_id, active_only, completed_only, offset, limit, version=version)
+    api_version = version or api_client.jobs_api_version
+    if api_version != '2.1' and (expand_tasks or offset or limit or _all):
+        click.echo(click.style('ERROR', fg='red') + ': the options --expand-tasks, ' +
+                   '--offset, --limit, and --all are only available in API 2.1', err=True)
+        return
```

Comment:
Why does the user need to care about the API version in the first place? If every workspace has both APIs, why don't we only use the 2.1 API in the CLI?

Reply:
We provide the option of choosing API version due to slight differences in the response structure.

Reply:
Of course, understood.